sw-build: Error: Cannot find module 'chalk'

**Library Affected**:
sw-build

**Browser & Platform**:
Node v7.7.2, Yarn v0.21.3

**Issue or Feature Request Description**:
When I run `require('sw-build')`, I get error: `Error: Cannot find module 'chalk'`

*When reporting bugs, please include relevant JavaScript Console logs and links to public URLs at which the issue could be reproduced.*

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/GoogleChrome/sw-helpers#readme",
   "dependencies": {
     "idb": "^2.0.0",
+    "mkdirp": "^0.5.1",
     "update-notifier": "^1.0.3"
   },
   "devDependencies": {

--- a/packages/sw-build/package.json
+++ b/packages/sw-build/package.json
@@ -28,6 +28,7 @@
   "homepage": "https://github.com/GoogleChrome/sw-helpers/tree/master/packages/sw-build",
   "dependencies": {
     "chalk": "^1.1.3",
+    "glob": "^7.1.1",
     "lodash.template": "^4.4.0",
     "mkdirp": "^0.5.1",
     "sw-lib": "^0.0.17"

--- a/packages/sw-build/package.json
+++ b/packages/sw-build/package.json
@@ -27,6 +27,9 @@
   "bugs": "https://github.com/googlechrome/sw-helpers/issues",
   "homepage": "https://github.com/GoogleChrome/sw-helpers/tree/master/packages/sw-build",
   "dependencies": {
+    "chalk": "^1.1.3",
+    "lodash.template": "^4.4.0",
+    "mkdirp": "^0.5.1",
     "sw-lib": "^0.0.17"
   },
   "devDependencies": {

--- a/packages/sw-cli/package.json
+++ b/packages/sw-cli/package.json
@@ -31,7 +31,6 @@
   "dependencies": {
     "chalk": "^1.1.3",
     "inquirer": "^2.0.0",
-    "lodash.template": "^4.4.0",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
     "sw-build": "^0.0.17",

